sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOIT.java

```diff
@@ -184,11 +184,11 @@ private void cancelIfNotTerminal(PipelineResult readResult, PipelineResult.State
 
     void setSourceOptions(String sourceOptions);
 
-    @Description("Kafka server address")
+    @Description("Kafka bootstrap server addresses")
     @Validation.Required
-    String getKafkaBootstrapServerAddress();
+    String getKafkaBootstrapServerAddresses();
```

Comment:
Good catch, thanks for making this clearer! :)

Reply:
👍